helm-charts/infisical/templates/frontend-deployment.yaml

```diff
@@ -9,65 +10,69 @@ metadata:
   labels:
     {{- include "infisical.frontend.labels" . | nindent 4 }}
 spec:
-  replicas: {{ .Values.frontend.replicaCount }}
+  replicas: {{ $frontend.replicaCount }}
   selector:
     matchLabels:
       {{- include "infisical.frontend.matchLabels" . | nindent 6 }}
   template:
     metadata: 
       labels:
         {{- include "infisical.frontend.matchLabels" . | nindent 8 }}
-      {{- with .Values.frontend.podAnnotations }}
+      {{- with $frontend.podAnnotations }}
       annotations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
     spec:
       containers:
-      - name: {{ template "infisical.name" . }}-{{ .Values.frontend.name }}
-        image: "{{ .Values.frontend.image.repository }}:{{ .Values.frontend.image.tag | default .Chart.AppVersion }}"
-        imagePullPolicy: {{ .Values.frontend.image.pullPolicy }}
+      - name: {{ template "infisical.name" . }}-{{ $frontend.name }}
+        image: "{{ $frontend.image.repository }}:{{ $frontend.image.tag | default .Chart.AppVersion }}"
+        imagePullPolicy: {{ $frontend.image.pullPolicy }}
         readinessProbe:
           httpGet:
             path: /
             port: 3000
           initialDelaySeconds: 10
           periodSeconds: 10
-        {{- if .Values.frontend.kubeSecretRef }}
+        {{- if $frontend.kubeSecretRef }}
```

Comment:
It looks like the user has to define a kubeSecretRef + define the environment variables in values.frontend.secrets ? I feel like if they are defining `values.frontend.secrets` then we should just put those in a secret and auto reference them. Thoughts?